ee/tabby-ui/app/(home)/components/search.tsx

```diff
@@ -187,10 +191,10 @@ export function SearchRenderer({}, ref: ForwardedRef<SearchRef>) {
             })
           }
         }
-      }, 1500)
+      }, 300)
     }
 
-    if (!isLoading) {
+    if (!isLoadingRef.current) {
       setShowStop(false)
     }
```

Comment:
It would be better if the useEffect returned a clearTimeout function.

Reply:
thanks, updated